src/track/text-track.js

```diff
@@ -40,6 +46,15 @@ const TextTrack: TextTrack = class TextTrack extends Track {
     return this._external;
   }
 
+  /**
+   * Getter for the CEA of the text track.
+   * @public
+   * @returns {boolean} - The kind of the text track.
```

Comment:
It is copied pasted wrong :)
```suggestion
   * @returns {boolean} - returns true if the text track has no language specified
```